lib/helpers/utils.js

```diff
@@ -1,6 +1,14 @@
 module.exports = {
   uppercaseFirstChar: function (str) {
+    var exclude = ['and'];
```

Comment:
I like how this is easily extendable, should we add `or`, `with` and a few others to this list